LiskHQ/lisk-desktop
# Improve user experience when blockchain client or service is indexing

### What was the problem?

This PR resolves #5524

### How was it solved?

- [x] Introduce page loader when service is indexing

### How was it tested?
- Ensure you run a local instance of core and service
- Restart both instance of core and service while you ensure the app is opened and it network is switched to the local network.
- When service and core is up and running you should see the indexing loading status show up and update as the progress proceeds.
